### PR TITLE
Updated readme and added new composer banner

### LINKED ISSUE #5931: Review and update SDK readme
Review the SDK readme for accuracy and update any information that requires amends. e.g. Channels list etc.

Also make updates as specified in this doc https://microsoft.sharepoint.com/:w:/r/teams/ConversationalAI785/_layouts/15/Doc.aspx?sourcedoc=%7B4CB418CD-3DB5-499C-8FC1-AB6E36EB2D86%7D&file=Updates%20for%20SDK%20readme.docx&action=default&mobileredirect=true&wdLOR=c69DD491D-4905-3F4C-AB98-1F124BFD0CCF&cid=11dc26ad-4ecc-4f8f-af1f-a26a16163621

### PR DESCRIPTION
**Pull request by garypretty**
Fixes #5931 

- Make introductory text / heading amends
- Add Composer banner
- Update channels list (added Python and additional adapters)
- Removed 'Preview ' labels from Composer / VA throughout
- Update preview list
- Added Composer / CLI / Webchat repos to the Issues and Feature requests section
- Updated contribution section to link to the contribution readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+With the [Bot Framework SDK](#bot-framework-sdk-v4), developers can build bots that converse free-form or with guided interactions including using simple text or rich cards that contain text, images, and action buttons.  
 
-Checkout the [Bot Framework ecosystem](#bot-framework-ecosystem) section to learn more about other tooling and services related to the Bot Framework SDK.
+Developers can model and build sophisticated conversation using their favorite programming languages including C#, JS, and Python or using [Bot Framework Composer](https://aka.ms/bfcomposer), an open-source, visual authoring canvas for developers and multi-disciplinary teams to design and build conversational experiences with Language Understanding, QnA Maker and sophisticated composition of bot replies (Language Generation).
```
Comment (sgellock) [inline README.md:9]: it feels a bit odd that we don't mention Java here.  It's a balance.  Java is in preview.  but, you can build bots with the Java SDK.  And later on in the quick links section we have a link to Java.  I think we should mention Java in this paragraph

File: README.md [hunk 2]
```diff
@@ -169,11 +168,9 @@ Botkit is part of Microsoft Bot Framework and is released under the [MIT Open So
 [106]:https://github.com/howdyai/botkit/tree/master/packages/botbuilder-adapter-twilio-sms#readme
 [107]:https://github.com/howdyai/botkit/tree/master/packages/botbuilder-adapter-web#readme
 
-### Bot Framework Solutions (Preview)
+### Bot Framework Virtual Assistant
```
Comment (sgellock) [inline README.md:171]: I think we should title this "Bot Framework Virtual Assistant Solution Accelerator".  We need to start the process of differentiating the solution effort from the service effort.

Comment (garypretty) [reply README.md]: Amended.

File: README.md (hunk 1 above)
Comment (garypretty) [reply README.md]: Java added.

**Pull request merged**